bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideProbeTargets.java

```diff
@@ -366,6 +366,11 @@ public boolean primaryIsBagsTarget() {
     }
 
     /**
+     * Determine if the primary guide star exists but is not a BAGS target.
+     */
+    public boolean primaryIsManual() { return primaryTarget.exists(manualTargets::contains); }
```

Comment:
Oh, good point. That's not a test case I had thought of.

If there is a manual guide star that is already in the list and no primary, then BAGS would run and create a new BAGS entry for the same manual guide star. If there is a primary, BAGS won't run at all, so no issue.

Not sure the best way to handle this, to be honest: if we designate that star as the BAGS star, then it would no longer be a manual target, and additionally would no longer be editable, which is problematic.

The other option would be to have the manual star that BAGS would have selected be marked as the primary, but not designate it the BAGS star, which seems confusing.

Otherwise, this may require some more in-depth processing of allowing the BAGS star to be in the list of manual targets, and be editable if it is there. Seems a bit complicated to be implementing right now.

I'm frankly not quite sure the best way to handle this right off.